react/Column.js

```diff
@@ -1,7 +1,11 @@
 import React from 'react'
+import { useCssHandles } from 'vtex.css-handles'
+
+const CSS_HANDLES = ['column']
 
 const Column = ({ children }) => {
-  return <div className="flex items-center">{children}</div>
+  const handles = useCssHandles(CSS_HANDLES)
+  return <div className={`${handles.column}`}>{children}</div>
```

Comment:
Could this just be 
```suggestion
  return <div className={handles.column}>{children}</div>
```
?

Reply:
Actually, I forgot to add "flex items-center" here